src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -477,6 +485,7 @@
       <AdditionalFiles Include="$(ProjectConfFilePath)" />
     </ItemGroup>
 
+    <Message Importance="normal" Text="Sonar: ($(MSBuildProjectFile)) Analysis configured successfully" />
```

Comment:
maybe give the path to `$(SonarQubeConfigPath)\SonarQubeAnalysisConfig.xml` in the log? it may help self-diagnosis and support